cupyx/scipy/signal/_signaltools.py

```diff
@@ -723,6 +723,120 @@ def lfilter(b, a, x, axis=-1, zi=None):
         return out
 
 
+def lfiltic(b, a, y, x=None, axis=-1):
```

Comment:
`scipy.signal.lfiltic` does not have `axis` parameter, so we don't have to have it. Any reason?

Reply:
It is to be consistent with respect to the `lfilter` signature. However, since SciPy does not support it, we can leave it commented and enable it again if SciPy supports it someday